google/cloud/aiplatform/docker_utils/run.py

```diff
@@ -195,13 +216,13 @@ def run_prediction_container(
             )
         elif not credential_path_on_host.exists() and not credential_from_adc_env:
             raise ValueError(f'credential_path does not exist: "{credential_path}".')
-    credential_mount_path = _DEFAULT_CONTAINER_CRED_KEY_PATH
-    volumes = (
-        [f"{credential_path_on_host}:{credential_mount_path}"]
-        if credential_path
-        else []
-    )
-    envs[_ADC_ENVIRONMENT_VARIABLE] = credential_mount_path
+        credential_mount_path = _DEFAULT_CONTAINER_CRED_KEY_PATH
+        volumes = volumes + (
+            [f"{credential_path_on_host}:{credential_mount_path}"]
+            if credential_path
```

Comment:
Is this if needed? It's already in the `if credential_path:` block.

Reply:
you're right. Done.